packages/test-helpers/src/test-helpers.ts

```diff
@@ -78,3 +78,25 @@ export const isSafari = (version = undefined): boolean => { // Indicates if this
  */
 export const isFirefox = (): boolean => (/firefox/i).test(navigator.userAgent);
 /* c8 ignore stop */
+
+// ResizeObserver loop error is considered benign as discussed in https://github.com/w3c/csswg-drafts/issues/5023
+// This module converts the error into a warning instead
+// Note that, mocha must be available in an imported context
+before(function () {
+  const originalOnError = window.onerror;
+  window.onerror = function (event, ...args) {
+    // Firefox: `ResizeObserver loop completed with undelivered notifications.`
+    // Chrome: `ResizeObserver loop limit exceeded`
+    // Safari: `ResizeObserver loop completed with undelivered notifications`
+    // Each browser logs a slightly different messages yet they all start with `ResizeObserver loop`
+    /* istanbul ignore else */
```

Comment:
let's update to the syntax of v7 code coverage tool.

Reply:
fixed.